specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```diff
@@ -0,0 +1,6113 @@
+      "type": "string",
+      "enum": [
+        "primary",
+        "secondary"
+      ],
+      "x-ms-enum": {
+        "name": "CertificateEntry",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "primary",
+            "name": "Primary",
+            "description": "The entry of primary certificate"
+          },
+          {
+            "value": "secondary",
+            "name": "Secondary",
+            "description": "The entry of secondary certificate"
+          }
+        ]
+      },
+      "required": true
+    },
+    "Filter": {
+      "in": "query",
+      "name": "$filter",
+      "description": "An expression on the resource type that selects the resources to be returned.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": false
+    },
+    "JobId": {
+      "in": "path",
+      "name": "jobId",
+      "description": "Unique ID of the job.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "maxLength": 255,
+      "pattern": "^[a-zA-Z0-9_-]*$",
+      "required": true
+    },
+    "ModuleName": {
+      "in": "path",
+      "name": "moduleName",
+      "description": "Name of the device module.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "OrderBy": {
+      "in": "query",
+      "name": "$orderby",
+      "description": "An expression that specify the order of the returned resources.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": false
+    },
+    "OrganizationId": {
+      "in": "path",
+      "name": "organizationId",
+      "description": "Unique ID for the organization.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "maxLength": 48,
+      "pattern": "^(?!-)[a-z0-9-]{1,48}[a-z0-9]$",
+      "required": true
+    },
+    "RelationshipId": {
+      "in": "path",
+      "name": "relationshipId",
+      "description": "Unique ID of a relationship between devices.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "ScheduledJobId": {
+      "in": "path",
+      "name": "scheduledJobId",
+      "description": "Unique ID of the scheduled job.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "maxLength": 255,
+      "pattern": "^[a-zA-Z0-9_-]*$",
+      "required": true
+    },
+    "Subdomain": {
+      "in": "path",
+      "name": "subdomain",
+      "description": "The application subdomain.",
+      "x-ms-parameter-location": "client",
+      "type": "string",
+      "required": true
+    },
+    "TelemetryName": {
+      "in": "path",
+      "name": "telemetryName",
+      "description": "Name of this device telemetry.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "Top": {
+      "in": "query",
+      "name": "$top",
```

Comment:
Don't use "$" prefix on parameter names.

Reply:
Removed $filter, $top, $orderby from July GA, will revise this and get preview on Sep.